### PR TITLE
Feature/#22 generate mock data services

### PR DESCRIPTION
**Pull request by begumerden**
- Add faker js dependency
- Profile data service returning fake data
- Organization data service returning fake data

### CODE REVIEW

File: src/scripts/services/OrganizationService.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    const organization = this.organizations.find(o => o.id === id)
+    if (!organization) {
+      throw new Error('Organization not found')
+    }
+    return organization
+  }
+
+  public getBy (skip: number, take: number): Organization[] {
+    if (skip < 0 || take < 0) {
+      throw new Error('values must be bigger than zero.')
```
Comment (ilhan-mstf) [inline src/scripts/services/OrganizationService.ts]:
```suggestion
      throw new Error(`skip(${skip}) and take(${take}) must be bigger than zero.`)
```

File: src/scripts/services/ProfileService.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    const profile = this.profiles.find(p => p.id === id)
+    if (!profile) {
+      throw new Error('Profile not found')
+    }
+    return profile
+  }
+
+  public getBy (skip: number, take: number): Profile[] {
+    if (skip < 0 || take < 0) {
+      throw new Error('values must be bigger than zero.')
```
Comment (ilhan-mstf) [inline src/scripts/services/ProfileService.ts]:
```suggestion
      throw new Error(`skip(${skip}) and take(${take}) must be bigger than zero.`)
```

File: src/scripts/services/ProfileService.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+    this.createProfiles()
+  }
+
+  public getAll (): Profile[] {
+    return this.profiles
+  }
+
+  public getById (id: number): Profile {
+    if (id < 0) {
+      throw new Error('id must be bigger than zero')
```
Comment (ilhan-mstf) [inline src/scripts/services/ProfileService.ts]:
```suggestion
      throw new Error(`id(${id}) must be bigger than zero`)
```

File: src/scripts/services/ProfileService.ts [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    profile.updatedAt = new Date()
+    this.profiles[idx] = profile
+    return profile
+  }
+
+  public deleteById (id: number) {
+    if (id < 0) {
+      throw new Error('id must be bigger than zero')
```
Comment (ilhan-mstf) [inline src/scripts/services/ProfileService.ts]:
```suggestion
      throw new Error(`id(${id}) must be bigger than zero`)
```

**Comment by ilhan-mstf** [comment]
I've suggested small changes in order to see the value of id or take on the error message.

**Comment by ilhan-mstf** [comment]
It should be ` instead of '.
https://eslint.org/docs/rules/no-template-curly-in-string

**Review by koraytaylan — APPROVED** [review]

**Review by ilhan-mstf — APPROVED** [review]

**Pull request merged**